README.md

```diff
@@ -65,19 +68,22 @@ Despite the fact we love most of Bedrock, we noticed there are some things we do
 
 ## Requirements
 
+* Composer v2
 * Basic knowledge about bash scripting, deployment with capistrano, npm packages, bundle, composer etc.
 * Vagrant ([marlin-vagrant](https://github.com/digitoimistodude/marlin-vagrant)) OR [osx-lemp-setup](https://github.com/digitoimistodude/osx-lemp-setup) but can be configured for MAMP or even Docker (in planning)
 * GitHub account
 * Unix-based OS (built for Mac OS X by default)
 * Access to staging and production servers that supports sftp and git
 * Projects located under $HOME/Projects
 * Git
-* PHP >= 5.6
-* Ruby >= 2.2
+* PHP >= 7.1
```

Comment:
This is only partially true, because when using `pivvenit/acf-pro-installer` PHP 7.2 is required. It might not be used by users outside of our agency, but for our devs we need that package and thus 7.2

Reply:
New phpdotenv requires [at least php 7.1](https://github.com/vlucas/phpdotenv/blob/b3eac5c7ac896e52deab4a99068e3f4ab12d9e56/composer.json#L19).

Reply:
Also now that I checked Roots and oscarotero/env also require it. So it's pretty much a must-have minimum.

Reply:
Agreed, my point was to say that we probably should state the 7.2 as minimum version instead of 7.1 :)

Reply:
Aah, thought you are commenting about the PHP version update in general... tired brain, sorry. Yes, 7.1 is directly from those composer packages, should be 7.2 for us.